src/AssemblyAI/Types/Error.cs

```diff
@@ -4,13 +4,13 @@
 
 namespace AssemblyAI;
 
-public class Error
+public record Error
 {
     /// <summary>
     /// Error message
     /// </summary>
     [JsonPropertyName("error")]
-    public string Error_ { get; init; }
+    public required string Error_ { get; init; }
```

Comment:
Can we rename `Error_` to `Message` or `ErrorMessage`?

Reply:
Yep, 2 options here:
1. add this file to the `.fernignore` and make the change manually
2. update your OpenAPI spec to use a different name for the property (while of course maintaining the original json property. name). This would be a breaking change so up to you if you consider safe / worth it

Reply:
can guide you through either one, just let me know!